github_discovery.py

```diff
@@ -249,8 +290,22 @@ def process_repo(**component):
       if file.name.startswith('values-'):
         env = re.match('values-([a-z0-9-]+)\\.y[a]?ml', file.name)[1]
         helm_environments.append(env)
+
+        # HEAT-223 Start : Read and collate data for allowlist from all environment specific values.yaml files.
+        ip_allow_list[file] = fetch_values_for_allowlist_key(get_file_yaml(repo, f"{monorepo_dir_suffix}helm_deploy/{file.name}"), allow_list_key) 
+        ip_allow_list_data.update({f"ip_allow_list_{env}": ip_allow_list[file]})
+        # HEAT-223 End : Read and collate data for allowlist from all environment specific values.yaml files.
+
     helm_default_values = get_file_yaml(repo, f"{monorepo_dir_suffix}helm_deploy/{c_name}/values.yaml")
+    
     if helm_default_values:
+
+      # HEAT-223 Start : Read and collate data for allowlist from default values.yaml files.   
+      default_values = "default"
```

Comment:
suggest that default - is not specific or helpful, use `values.yaml` the name of the file as the key in your data.

Reply:
Code on 295 is reading all values files except the default one and fetch the payload. Code after 299 is under if condition is reading default values file only if there is one. This is the existing way that is implemented to read default values file and other environment specific files.

Reply:
Done.